### PR TITLE
HPCC-20065 Add sr translations

### PR DESCRIPTION
**Pull request by kunalaswani**
Adding sr translations and alphabetizing hpcc.js file

Signed-off-by: Kunal Aswani <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by kunalaswani** [comment]
@emuharemagic please review

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 1]
```diff
@@ -216,11 +234,12 @@
     FilePath: "Локација Датотеке",
     FilePermission: "Дозвола за приступ фајлу",
     Files: "Датотеке",
-    FilesPending: "Датотеке у току",
     FileScopeDefaultPermissions: "Предефинисанe дозволе за простор Фајлова",
     FileScopes: "Скоп Датотека",
     FileSize: "Величина Датотеке",
     FilesNoPackage: "Датотеке које не припадају пакету",
+    FilesPending: "Датотеке у току",
+    FilesWarning: "Broj vraćenih datoteka je preveliki. Vraćeno je samo prvih 100.000 datoteka sortiranih po datumu / vremenu. Ako želite ograničiti rezultate, podesite filter.",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 2]
```diff
@@ -248,6 +267,7 @@
     GetPart: "Добавите Део",
     GetSoftwareInformation: "Желите ли да добијетe информације о софтверу",
     Graph: "Граф",
+    GraphControl: "Kontrola Grafova",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 3]
```diff
@@ -295,6 +315,9 @@
     Largest: "Највећи",
     LargestFile: "Највећа Датотека",
     LargestSize: "Највећа величина",
+    LastEdit: "Poslednja Izmjena",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 4]
```diff
@@ -295,6 +315,9 @@
...
+    LastEditedBy: "Autor Poslednje Izmjene",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 5]
```diff
@@ -295,6 +315,9 @@
...
+    LastEditTime: "Vrijeme Poslednje Izmjene",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 6]
```diff
@@ -303,6 +326,7 @@
     LastRun: "Задњи Ран",
     LDAPWarning: "<б>Грешка ЛДАП Сервица:</б>  &lsquo;Превише корисника&rsquo; - Молимо користите филтер.",
     LegacyForm: "Стари Превазиђени Формулар",
+    Legend: "Legenda",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 7]
```diff
@@ -317,19 +341,24 @@
     Local: "Локални",
     LocalFileSystemsOnly: "Само Локални Фајл Системи",
     Location: "Локација",
+    Lock: "Zaključaj",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 8]
```diff
@@ -317,19 +341,24 @@
...
+    log_analysis_1: "лог_аналисис_1*",
     Log: "Дневник (Лог)",
     LogFile: "Датотека Активности",
     LoggedInAs: "Пријављен као",
+    LoggingOut: "Odjavljivanje",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 9]
```diff
@@ -317,19 +341,24 @@
...
     LogicalFile: "Логичка Датотека",
     LogicalFiles: "Логичке Датотеке",
     LogicalFilesAndSuperfiles: "Логичке Датотеке и Супердатотеке",
     LogicalFilesOnly: "Само Логичке Датотеке",
     LogicalFileType: "Тип Логичке Датотеке",
     LogicalName: "Име Логичке Датотеке",
+    Login: "Prijavljivanje",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 10]
```diff
@@ -317,19 +341,24 @@
...
+    Logout: "Odjavite se",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 11]
```diff
@@ -317,19 +341,24 @@
...
     Logs: "Дневници",
-    log_analysis_1: "лог_аналисис_1*",
-    LostFile2: "Изгубљени Фајлови",
+    LogVisualization: "Registrujte Vizuelizaciju",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 12]
```diff
@@ -347,15 +376,18 @@
     MaxNode: "Максимални НОД / чвор",
     MaxRecordLength: "Максимална Дужина Рекорда",
     MaxSize: "Максимална величина",
+    MaxSkew: "Maksimalni Skju (Distorzija)",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 13]
```diff
@@ -347,15 +376,18 @@
...
     Message: "Поруке",
-    Methods: "Методи",
     MethodConfiguration: "Конфигурација метода",
+    Methods: "Методи",
     Min: "Минимум",
+    Mine: "Moj",
     MinNode: "Минимални НОД / чвор",
     Minor: "Миноран",
     MinSize: "Минимална Величина",
+    MinSkew: "Minimalni Skju (Distorzija)",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 14]
```diff
@@ -455,9 +487,11 @@
     PlaceholderLastName: "Смит",
     Playground: "Игралиште",
     PleaseEnterANumber: "Унестите Број 1 -",
+    PleaseLogin: "Molimo prijavite se koristeći svoje korisničko ime i lozinku",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 15]
```diff
@@ -455,9 +487,11 @@
...
     PleasePickADefinition: "Изаберите дефиницију",
     PleaseSelectADynamicESDLService: "Одаберите Динамички ЕСДЛ сервис",
     PleaseSelectAGroupToAddUser: "Изаберите групу у коју желите да додате корисника",
+    PleaseSelectAServiceToBind: "Izaberite Servis za Upotrebu",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 16]
```diff
@@ -484,6 +518,7 @@
     Publish: "Објавите",
     Published: "Објављен",
     PublishedBy: "Објављивач",
+    PublishedByMe: "Moji Objavljeni Servisi",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 17]
```diff
@@ -502,13 +537,15 @@
     RawTextPage: "Необрађен Текст (Текућа Страница)",
     Ready: "Cпреман",
     ReallyWantToRemove: "Заиста желите уклонити?",
+    ReAuthenticate: "Ponovo se prijavite koristeći svoje korisničko ime i lozinku",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 18]
```diff
@@ -522,15 +559,17 @@
...
     RemoveSubfiles: "Уклоните Поддатотеку",
+    RemoveSubfiles2: "Желите ли да уклоните субфајл(ове)?",
     RemoveUser: "Уклоните Корисника",
     Rename: "Преименујте",
     RenderedSVG: "Донесене СВГ",
     RenderSVG: "Донесите СВГ",
     Replicate: "Реплицирајте",
     ReplicatedLost: "Изгубљене Репликације",
     ReplicateOffset: "Реплицирајте Офсет",
+    ReportAnError: "Prijavite grešku",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 19]
```diff
@@ -522,15 +559,17 @@
...
+    ReportError: "Prijavite grešku",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 20]
```diff
@@ -569,6 +608,7 @@
     SelectPackageFile: "Изаберитe Пакет",
     Separators: "Сеператори",
     Server: "Сервер",
+    ServiceName: "Ime Servisa",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 21]
```diff
@@ -631,6 +671,7 @@
     SVGSource: "СВГ Извор",
     SyncSelection: "Синхронизујте Са Одабраним",
     SystemServers: "Систем Сервера",
+    Table: "Tabela",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 22]
```diff
@@ -660,10 +701,12 @@
     title_Activity: "Активности",
     title_AvailableGroupPermissions: "Доступне Дозволе групе",
     title_AvailablePermissions: "Расположиве Дозволе",
+    title_BindingConfiguration: "Konfiguracija za Povezivanje",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 23]
```diff
@@ -660,10 +701,12 @@
...
+    title_BindingDefinition: "Definicija Povezivanja",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 24]
```diff
@@ -732,14 +775,17 @@
     TransitionGuide: "Водич",
     Tree: "Дрво",
     Type: "Тип",
+    Unbound: "Odvojen",
     undefined: "недефинисан",
     Unknown: "Непознат",
+    Unlock: "Otključajte",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 25]
```diff
@@ -732,14 +775,17 @@
...
     Unprotect: "Укините Заштиту",
     UnsupportedIE9FF: "Нису Подржани (ИЕ <= 9, ФиреФоx)",
     Unsuspend: "Укините Суспензију",
     Unsuspended: "Суспензија Укинута",
     Up: "На Горе",
     UpdateCloneFrom: "Ажурирајте Клон Користећи",
+    UpdateDFs: "Ažurirajte DFS",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 26]
```diff
@@ -748,6 +794,7 @@
     User: "Корисник",
     UserDetails: "Детаљи о Кориснику",
     UserID: "Корисницки ИД",
+    UserLogin: "Prijavite se samo koristeći svoje korisničko ime",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 27]
```diff
@@ -793,6 +840,7 @@
     XRef: "XRef",
     Year: "Година",
     YouAreAboutToDeleteBinding: "Одабрано везивање ће бити обрисано. Да ли сте сигурни да желите да то урадите?",
+    YouAreAboutToDeleteDefinition: "Vi ste u procesu brisanja ove definicije. Da li ste sigurni da to želite učiniti?",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: pls use the correct source tab for this

**Review by emuharemagic — CHANGES_REQUESTED** [review]
looks like most of the changes were picked up from the wrong source;

**Comment by kunalaswani** [comment]
@emuharemagic sorry about that, my tabs got crossed, please review, corrected.

**Review by emuharemagic — APPROVED** [review]

**Pull request merged**